app/views/resources/show.html+teacher.erb

```diff
@@ -34,6 +34,12 @@
             <div class="card-header">
               <strong><%= m.start_time.strftime("%a %-m/%-d/%y - %l:%M%P") %></strong>
             </div>
+            <div class="card-body">
+              <div class="panel-group">
+                <%= render partial: "/attendances/attendance_random_student_picker",
+                                   locals: {student: m.attendances.accepted.sample} %>
```

Comment:
@pmckernin This is conceptually clear and gets the random student we want. After playing around with it, I think it might be better to exclude a student who was just picked from the next random selection just in case someone is very unlucky or if a class size happens to be particularly small.

Reply:
I agree with this idea; we should keep track of who has been picked, and only choose from those who have not been picked, until everyone has been picked. Then a new "round" can begin.